catroid/src/test/java/org/catrobat/catroid/test/web/CatrobatWebClientTest.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2022 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.web
+
+import okhttp3.Call
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import okhttp3.ResponseBody
+import org.catrobat.catroid.web.WebconnectionException
+import org.catrobat.catroid.web.performCallWith
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.powermock.api.mockito.PowerMockito
+import org.powermock.core.classloader.annotations.PrepareForTest
+import org.powermock.modules.junit4.PowerMockRunner
+import java.io.IOException
+
+@RunWith(PowerMockRunner::class)
+@PrepareForTest(Request::class, Response::class, ResponseBody::class)
+class CatrobatWebClientTest {
+    @Rule
+    val exception = ExpectedException.none()
+    private var requestMock: Request? = null
```

Comment:
use `lateinit` for non-null properties that will be initialized later. This way, you can avoid using nullable types and the not-null assertion operator (!!)

```suggestion
    private lateinit var requestMock: Request
```